_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -126,26 +126,26 @@ The `aria-checked` [state][] is [required][] for the [semantic][semantic role] `
 
 #### Passed Example 6
 
-The `aria-controls` [property][] is [required][] for the [semantic][semantic role] `combobox`.
+The `aria-expanded` [property][] is [required][] for the [semantic][semantic role] `combobox`.
 
 ```html
 <div role="combobox" aria-controls="id1" aria-expanded="false">My combobox</div>
 ```
 
 #### Passed Example 7
 
-The `aria-controls` [property][] is [required][] for the [semantic][semantic role] `combobox`. [WAI-ARIA states and properties][wai-aria state or property] with empty value are still applicable to this rule.
+The `aria-expanded` [property][] is [required][] for the [semantic][semantic role] `combobox`.
 
 ```html
-<div role="combobox" aria-expanded="false" aria-controls>My combobox</div>
+<div role="combobox" aria-expanded="true" aria-controls="id1">My combobox</div>
```

Comment:
This one feels off. If this is an expanded combobox it would fail for not having a popup.

```suggestion
<div role="combobox" aria-expanded="false" aria-controls="id1">My combobox</div>
```